src/cli/cli.cpp

```diff
@@ -5720,7 +5783,35 @@ template <> otError Interpreter::Process<Cmd("panid")>(Arg aArgs[])
 template <> otError Interpreter::Process<Cmd("parent")>(Arg aArgs[])
 {
     otError error = OT_ERROR_NONE;
-
+    /**
+     * @cli parent
+     * @code
+     * parent
+     * Ext Addr: be1857c6c21dce55
+     * Rloc: 5c00
+     * Link Quality In: 3
+     * Link Quality Out: 3
+     * Age: 20
+     * Version: 4
+     * Done
+     * @endcode
+     * @code
+     * CSL clock accuracy: 20
+     * CSL uncertainty: 5
+     * @endcode
```

Comment:
This CSL part is not an example of the command usage on its own, it's only relevant to the note further below.  You can remove this.